nfs-ganesha/config/definitions/nfs-ganesha.yml

```diff
@@ -114,16 +123,16 @@ If this is checked, then the binaries will be built and pushed to chacra even if
       - pollurl:
           cron: '*/30 * * * *'
           urls:
-            - url: https://shaman.ceph.com/api/repos/ceph/master/latest/centos/7/repo
+            - url: https://shaman.ceph.com/api/repos/ceph/{scenario}/latest/centos/7/repo
```

Comment:
The job template is rendered for each additional field you've defined, in this case ``scenario``. This means that you'll have two duplicate polling urls here. The same problem exists for the urls looking for new xenial repos.